modules/renter/downloadchunk.go

```diff
@@ -22,6 +22,13 @@ type downloadPieceInfo struct {
 	root  crypto.Hash
 }
 
+// cacheData contatins the data and the timestamp for the unfinished
+// download chunks
+type cacheData struct {
```

Comment:
We usually try to sort declarations of structs, functions and fields. In this case `cacheData` should be declared above `downloadPieceInfo`